framework/db/mysql/QueryBuilder.php

```diff
@@ -147,6 +147,18 @@ public function checkIntegrity($check = true, $schema = '', $table = '')
     /**
      * @inheritdoc
      */
+    public function buildFrom($tables, &$params)
+    {
+        if (count($tables) < 2) {
```

Comment:
While this is true, it just feels unnatural and unnecessary to have extra parenthesis. It's like you always add parenthesis to the SELECT part or other parts.